<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>心跳点名向导</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>输入账号</string>
   </property>
   <property name="subTitle">
    <string>检测到你还没有创建使用数据，请一步一步开始创建吧！</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>班级名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="classNameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>管理密码</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>学生名单管理</string>
   </property>
   <property name="subTitle">
    <string>请确保每一行只包含一个姓名</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="tipsLabel">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
